src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAuthenticationManager.java

```diff
@@ -298,7 +298,7 @@ public RegisteredUser getSegueUserFromLinkedAccount(final AuthenticationProvider
      */
```

Comment:
The documentation needs reviewing. Could you go through all the methods that you've changed and update the documentation, please?